python/servo/build_commands.py

```diff
@@ -315,11 +315,15 @@ def build(self, release=False, dev=False, jobs=None, params=None, media_stack=No
             process = subprocess.Popen('("%s" %s > nul) && "python" -c "import os; print(repr(os.environ))"' %
                                        (os.path.join(vs_dirs['vcdir'], "Auxiliary", "Build", "vcvarsall.bat"), "x64"),
                                        stdout=subprocess.PIPE, shell=True)
-            stdout, _ = process.communicate()
+            stdout, err = process.communicate()
```

Comment:
I think this should be stderr?

Reply:
Oops, i'd manually copied over the patch